src/components/utils/NameLink.js

```diff
@@ -38,7 +38,7 @@ const NameLink = React.forwardRef((props, ref) => {
                 }
             }}
         >
-            <Highlighter search={searchTerm}>{name}</Highlighter>
+            <Highlighter search={searchTerm || ""}>{name}</Highlighter>
```

Comment:
Without this change a warning was issued if the `searchTerm` prop was not set in `NameLink`, although `Highlighter` appears to display its child components just fine (like a no-op) with an empty `search` prop.